.github/workflows/build.yml

```diff
@@ -42,14 +42,14 @@ jobs:
           mkdir -p ~/download
           cd ~/download
           wget https://cdn.discordapp.com/attachments/698589325620936736/845499146982129684/mwccarm.zip
-          wget https://cdn.discordapp.com/attachments/698589325620936736/722822401963851797/NitroSDK-3_2-060901.7z
+          wget https://cdn.discordapp.com/attachments/855279572651868170/875195363261292604/NitroSDK-4_2-071210-jp.7z
           unzip mwccarm.zip
           mv mwccarm $GITHUB_WORKSPACE/tools
-          7z x NitroSDK-3_2-060901.7z
-          mv NitroSDK-3_2-060901/tools/bin $GITHUB_WORKSPACE/tools
-          mv NitroSDK-3_2-060901/include/nitro/specfiles/ARM7-TS.lcf.template $GITHUB_WORKSPACE/sub/
-          mv NitroSDK-3_2-060901/include/nitro/specfiles/ARM9-TS.lcf.template $GITHUB_WORKSPACE/
-          mv NitroSDK-3_2-060901/include/nitro/specfiles/mwldarm.response.template $GITHUB_WORKSPACE/
+          7z x NitroSDK-4_2-071210-jp.7z
+          mv NitroSDK-4_2-071210-jp/tools/bin $GITHUB_WORKSPACE/tools
```

Comment:
The build failed here. Looks like the 4.2 zip doesn't include the enclosing `NitroSDK-4_2-071210-jp` directory at the root like the 3.2 zip does.

Reply:
Should be fixed with this new commit, but haven’t tested.